pkg/resources/mutation/cluster/cluster.go

```diff
@@ -108,7 +108,7 @@ func (m *ManagementClusterMutator) setPSAConfig(cluster *apisv3.Cluster) error {
 	if err != nil {
 		return fmt.Errorf("failed to get PodSecurityAdmissionConfigurationTemplate: %w", err)
 	}
-	plugin, err := psa.GetPluginConfigFromTemplate(template)
+	plugin, err := psa.GetPluginConfigFromTemplate(template, cluster.Spec.RancherKubernetesEngineConfig.Version)
```

Comment:
Can `RancherKubernetesEngineConfig` ever be nil?

Reply:
it is impossible because the `Admit` function has returned before calling this function if Cluster.Spec.RancherKubernetesEngineConfig == nil